src/main/java/com/gamegoo/gamegoo_v2/content/board/service/BoardFacadeService.java

```diff
@@ -52,4 +61,25 @@ public BoardResponse getBoardList(Integer mode, Tier tier, Integer mainPosition,
         return BoardResponse.of(boardPage);
     }
 
+    /**
+     * 회원 게시판 글 단건 조회 (파사드)
+     * - “회원 전용” 조회 로직
+     */
+    public BoardByIdResponseForMember getBoardByIdForMember(Long boardId, Long memberId) {
+
+        Board board = boardService.findBoard(boardId);
+
+        if (board.isDeleted()) {
+            throw new BoardException(ErrorCode.BOARD_NOT_FOUND);
+        }
```

Comment:
여기서 삭제됐을때 에러처리를 facade에서 하지 않고, boardService 로직에서 deleted가 false인 board만 조회하게 하는건 어떤가요..?!
```suggestion
        boardRepository.findByIdAndDeleted(boardId,false).orElseThrow(() -> new BoardException(ErrorCode.BOARD_NOT_FOUND));
```
boardService에서 이렇게 해주면 굳이 facade에서 에러처리를 하지 않아도 되어서요..!